app/controllers/posts_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
 
```

Comment:
- [ ] Your app solves N+1 problem when fetching all posts and their comments for a user as shown in the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLAZ4K2ZHA777536NECDBHORQS) Example in the posts#index actions you should use `@posts = @user.posts.includes(:comments)` instead of `@posts = @user.posts`